src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -498,11 +536,11 @@ def send_comm_message(self, msg_type: str, content: dict) -> None:
         self._comm.send(msg)
 
     def send_error_message(
-        self, err_type: str, req: Union[JobRequest, dict, str], content: dict = None
+        self, req: Union[JobRequest, dict, str], content: dict = None
```

Comment:
there's only one type of error message (`job_error`) so there's no point in having the `err_type` parameter any more